index.html

```diff
@@ -10,7 +10,51 @@
     <link rel="stylesheet" href="styles/index.css" />
 </head>
 <body>
-    <script src="./src/index.js"></script>
+    <header>
+        <h1>Weather Report</h1>
+        <span>
+            Location:
+            <span id="cityLocation" class="city">Seattle</span>
+        </span>
```

Comment:
```suggestion
        <p>
            Location:
            <span id="cityLocation" class="city">Seattle</span>
        </p>
```
I'd recommend a `<p>` tag here instead for the semantic meaning. `<span>` tags should be used sparingly as they're meant for inline styling or class/id assignment. How you used the `<span>` tag for the city location is perfect though!